### DISCUSSION TITLE
put a shorter invitation link for the servers

### DISCUSSION THREAD

**Discussion by Zarkness2** [stoatchat/discussions#267] [Feature Suggestions]
put a shorter invitation link to the servers, something like revolt.gg/Wsx2Adsf or revolt.chat/Wsx2Adsf instead of links like https://app.revolt.chat/invite/Wsx2Adsf.

**Comment by sussycatgirl** [comment]
https://rvlt.gg/$INVITECODE already exists :)

**Comment by Zarkness2** [comment]
but when I create an invitation I don't get this invitation, I get this https://app.revolt.chat/invite/VpdJMeV4

**Reply by Rexogamer** [reply]
You have to manually "create" them - it's as simple as copying the code and pasting it at the end of the URL (in your case, `https://rvlt.gg/VpdJMeV4`).
